pnpm-lock.yaml

```diff
@@ -6506,7 +6508,6 @@ packages:
     cpu: [arm]
     os: [android]
     requiresBuild: true
-    dev: true
```

Comment:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/40715044/223333424-8fe064e0-dec4-42a3-a0aa-cd4a7bf0627a.png">

I don't know why, but my `pnpm@7.24.3` always does this.